packages/docs/docs/tutorials/data-modeling/family-relationships.md

```diff
@@ -0,0 +1,278 @@
+| :-: | :----------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------- | --------------------------------------------- | -------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------- |
+|  1  | [**Person**](/docs/api/fhir/resources/patient) with Contact Information                                                                                      | No                                                    | Rarely                                        | Only Patient                                                                                                   | Individual                                                            |
+|  2  | [**Person**](/docs/api/fhir/resources/patient)+ [**RelatedPerson**](/docs/api/fhir/resources/relatedperson)                                                  | Yes                                                   | Rarely                                        | Only Patient                                                                                                   | Individual                                                            |
+|  3  | [**Patient**](/docs/api/fhir/resources/patient)/ [**RelatedPerson**](/docs/api/fhir/resources/relatedperson) /[**Person**](/docs/api/fhir/resources/person)  | Yes                                                   | Frequently                                    | Only Patient                                                                                                   | Individual                                                            |
+|  4  | [**Patient**](/docs/api/fhir/resources/patient)/ [**RelatedPerson**](/docs/api/fhir/resources/relatedperson) /[**Person**](/docs/api/fhir/resources/patient) | Yes                                                   | Frequently                                    | All Members                                                                                                    | Individual                                                            |
+|  5  | [**Group**](/docs/api/fhir/resources/group) of Patients                                                                                                      | Yes                                                   | Frequently                                    | All Members                                                                                                    | Family Unit                                                           |
+
```

Comment:
Should give special treatment to minors here, and call out what actions if any they need to take.  Are the minors logging in and viewing their own records?  Getting notified?